src/test/java/org/folio/rest/api/MigrationTestBase.java

```diff
@@ -113,9 +116,24 @@ UpdateResult unsetJsonbProperty(String tableName, UUID id, String propertyName)
       resultSet -> {
         if (resultSet.failed()) {
           result.completeExceptionally(resultSet.cause());
-        } else {
-          result.complete(resultSet.result().getResults());
+          return;
+        }
+        RowIterator<Row> iterator = resultSet.result().iterator();
+        List<JsonArray> list = new LinkedList<>();
+        while (iterator.hasNext()) {
+          Row row = iterator.next();
+          JsonArray ar = new JsonArray();
+          for (int i = 0; i < row.size(); i++) {
+            Object obj = row.getValue(i);
+            if (obj instanceof java.util.UUID) {
+              ar.add(obj.toString());
+            } else {
+              ar.add(obj);
+            }
+          }
+          list.add(ar);
```

Comment:
Is this basically the same code as above?

Reply:
Most likely, will refactor

Reply:
Done